sopel/db.py

```diff
@@ -98,13 +99,21 @@ def __init__(self, config):
         # Handle SQLite explicitly as a default
         if db_type == 'sqlite':
             path = config.core.db_filename
-            config_dir, config_file = os.path.split(config.filename)
+            _, config_file = os.path.split(config.filename)
```

Comment:
You can actually replace all of that by using `config.basename`.

See https://github.com/sopel-irc/sopel/blob/master/sopel/config/__init__.py#L109

Reply:
:+1: